Update repos for an Org Team

marcohutzsch1234 opened the pull request:
Thanks your your amazing work for creating this node module to easily rename branches in a bunch of repositorys.

This PR makes it easier to rename master  -> main branche only in repositories which are linked to a team in an organization. 

If parameters org and team are provided all repos an existing team
are being looked up and updated. There is no functional change
on the existing org or user parameters.

mheap reviewed (changes requested):
Thanks for the PR!

It looks as though we'll fetch all repos for an org even if the `team` flag is provided (https://github.com/marcohutzsch1234/github-default-branch/blob/main/src/get-repos.js#L7-L16)

Could you refactor so that `listForOrg` is only called if `args.team` is not provided?

marcohutzsch1234 commented:
Hi @mheap you are right, I did not exclude the full org lookup even if the team is provided. I've added the args.team condition for the listForOrg as well, so it should work now. Code does not look really DRY, but its still good readable.

I was thinking of adding the --team parameter into the front validation but decided to leave the front validation as it is, because it would only complicate stuff instead of making anything clearer.

I squashed the change into the existing commit, the PR is updated.

mheap commented:
Thanks for this @marcohutzsch1234 - it's now released as `github-default-branch` v1.9.0
